tests/telemetry/test_telemetry.py

```diff
@@ -133,3 +157,15 @@ def test_telemetry_ouput(duthosts, rand_one_dut_hostname, ptfhost, localhost):
     result = str(show_gnmi_out)
     inerrors_match = re.search("SAI_PORT_STAT_IF_IN_ERRORS", result)
     pytest_assert(inerrors_match is not None, "SAI_PORT_STAT_IF_IN_ERRORS not found in gnmi_output")
+
+def test_osbuild_version(duthosts, rand_one_dut_hostname, ptfhost, localhost):
+    """ Test osbuild/version query.
+    """
+    duthost = duthosts[rand_one_dut_hostname]
+    cmd = generate_client_cli(duthost=duthost, mode=mode_get, target="OTHERS", xpath="osversion/build")
+    logger.info("Command to run: {0}".format(cmd))
+    show_gnmi_out = ptfhost.shell(cmd)['stdout']
+    logger.info(show_gnmi_out)
```

Comment:
Current infrastructure can log the executed command and execution output at DEBUG level. Is it necessary to log these information at INFO level again?

Reply:
Ack will change to debug